src/openrct2/network/DiscordService.cpp

```diff
@@ -24,7 +24,7 @@
 
 constexpr const char* APPLICATION_ID = "378612438200877056";
 constexpr const char* STEAM_APP_ID = nullptr;
-constexpr const uint32_t REFRESH_INTERVAL = 5 * GAME_UPDATE_FPS; // 5 seconds
+constexpr const float REFRESH_INTERVAL = 5.0f; // seconds
```

Comment:
Have you tried changing this from float to `std::chrono::duration`? We could use the standard defined literals for duration values, then. [`5s` instead of `5.0f`](https://en.cppreference.com/w/cpp/chrono/operator%22%22s).